index.js

```diff
@@ -0,0 +1,87 @@
+const generateBtn = document.querySelector("#generate");
```

Comment:
Would have been good to include this file in the `assets` folder within a `js` folder to keep the folder structure consistent.